src/main/java/com/axibase/tsd/client/SelfSignedTrustManager.java

```diff
@@ -1,34 +1,61 @@
 package com.axibase.tsd.client;
 
+import lombok.extern.slf4j.Slf4j;
+
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.TrustManagerFactory;
 import javax.net.ssl.X509TrustManager;
+import java.io.IOException;
+import java.io.Serializable;
+import java.net.Socket;
 import java.security.KeyStore;
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
 
+@Slf4j
+public class SelfSignedTrustManager implements X509TrustManager, Serializable, Cloneable {
 
-public class IgnoringTrustManager implements X509TrustManager {
-
-    public IgnoringTrustManager() throws NoSuchAlgorithmException, KeyStoreException {
+    public SelfSignedTrustManager() throws NoSuchAlgorithmException, KeyStoreException {
     }
 
     @Override
     public X509Certificate[] getAcceptedIssuers() {
         return new X509Certificate[0];
     }
 
-    @Override
     public void checkServerTrusted(X509Certificate[] chain,
                                    String authType) throws CertificateException {
-        //No implementation here
+        Socket socket = new Socket();
+        try {
+            socket.shutdownInput();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS1148)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Method com.axibase.tsd.client.SelfSignedTrustManager.checkServerTrusted(X509Certificate[], String) prints the stack trace to the console [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=fb-contrib%3AIMC_IMMATURE_CLASS_PRINTSTACKTRACE)